packages/cli/openapi-ir-to-fern/src/__test__/__snapshots__/intercom.test.ts.snap

```diff
@@ -37490,7 +37527,7 @@ If you are redacting a conversation part, it must have a \`body\`. If you are re
               {
                 "name": "User reply",
                 "path-parameters": {
-                  "id": "id",
+                  "id": "123 or "last"",
```

Comment:
This doesn't seem right

Reply:
i bet thats right -- we should escape string examples. I wonder if there is a single place in `openapi-ir-to-fern` where this could happen

Reply:
https://github.com/fern-api/fern/blob/07161c212c5edf340cf5f693d3dda0bf5854bdbf/packages/cli/openapi-ir-to-fern/src/utils/convertFullExample.ts#L44-L48

Reply:
it's likely because jest isn't configured to escape strings: https://jestjs.io/blog/2022/08/25/jest-29

Reply:
seems ok in IR: 
<img width="924" alt="image" src="https://github.com/user-attachments/assets/40705ff8-4b9b-4235-b1b0-196d1892c4bd">
Assuming that the reason is the jest reason @abvthecity mentioned